repair/row_level.cc

```diff
@@ -496,7 +496,7 @@ class repair_writer {
         };
         auto& db = service::get_local_storage_service().db();
         table& t = db.local().find_column_family(_schema->id());
-        _writer_done[node_idx] = mutation_writer::distribute_reader_and_consume_on_shards(_schema, dht::global_partitioner(),
+        _writer_done[node_idx] = mutation_writer::distribute_reader_and_consume_on_shards(_schema, _schema->get_partitioner(),
```

Comment:
Again, schema + partitioner passed to a method.

Reply:
fixed.